actors.md

```diff
@@ -708,3 +708,233 @@ func UpdatePeerID(pid PeerID) {
 ## Payment Channel Broker Actor
 
 TODO
+
+## Multisig Account Actor
```

Comment:
How do I create one of those actors?

Reply:
Using the init actor.

- [ ] detail how this works